src/rogue/protocols/packetizer/Controller.cpp

```diff
@@ -62,7 +62,9 @@ rpp::Controller::Controller ( rpp::TransportPtr tran, rpp::ApplicationPtr * app,
 }
 
 //! Destructor
-rpp::Controller::~Controller() { }
+rpp::Controller::~Controller() {
+   this->stopQueue();
```

Comment:
There is a `stopQueue()` and `stop()` for this module.  Should the `stop()` be used in the destructor instead?  This assumes that `stopQueue()` is executed inside `stop()`

Reply:
I almost went down that road, but this is an inner class and the cleanup is handled at a higher level. so stopQueue() is just an internal call. It also does not have a stop() call.